gammapy/spectrum/models.py

```diff
@@ -879,7 +879,33 @@ def to_sherpa(self, name='default'):
         name : str, optional
             Name of the sherpa model instance
         """
-        from .sherpa_utils import SherpaExponentialCutoffPowerLaw
+        from sherpa.models import ArithmeticModel, modelCacher1d, Parameter
+        
+        class SherpaExponentialCutoffPowerLaw(ArithmeticModel):
```

Comment:
I don't know if this one is specifically needed. This is for sherpa users who want to use an exponential cutoff PL model, right?
If we get rid of sherpa in most of the places, it is better to leave this do tests fully outside gammapy, no? I don't know how users could access this but does it really belong to gammapy/spectrum/models?

Reply:
Note that there's already this model here: https://github.com/zblz/naima/blob/master/naima/sherpa_models.py#L74

I would prefer to delete Sherpa models from Gammapy for now, and focus on developing our modeling.

Then we can still add things back to interface with Astropy or Sherpa, either for specific models or in a generic way like http://saba.readthedocs.io/en/latest/

Reply:
@registerrier I left this only for you :laughing: I thought you like to fit our data directly with sherpa. I'm happy to remove it. At the moment all our spectralmodel have a ``to_sherpa`` function, which returns an equivalent sherpa model. So I will get rid of this method for all spectral models ok?

Reply:
:+1: to remove all the `to_sherpa()` methods.